src/components/WidgetList/WidgetList.js

```diff
@@ -12,31 +12,57 @@ const WidgetList = props =>
   <div className="WidgetList">
     <div className="row">
       <div className="col-xs-12 col-sm-6 col-md-3">
-        <Number title="Week" data={props.weekNumber} />
+        <Number heading="Week" data={props.weekNumber} />
       </div>
       <div className="col-xs-12 col-sm-6 col-md-3">
-        <Number title="Load Time" data={props.newRelicLoadTime} riseIsBad threshold={0.33} />
+        <Number
+          heading="Load Time (s)"
+          description="since 10 minutes ago"
+          data={props.newRelicLoadTime}
+          riseIsBad
+          threshold={0.33}
+        />
       </div>
       <div className="col-xs-12 col-sm-6 col-md-3">
-        <Number title="Transaction Errors" data={{ current: 1, stuff: props.newRelicErrors }} riseIsBad threshold={1} />
+        <Number
+          heading="Transaction Errors"
+          description="since 30 minutes ago compare with 30 minutes ago"
```

Comment:
I think the description should be passed from the backend because only there you know the query.